src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -284,11 +282,13 @@ class LovelacePanel extends LitElement {
       }
     }
 
-    this._state = "loaded";
-    this._setLovelaceConfig(conf, confMode);
-    if (resources) {
+    if (resourceProm) {
+      const resources = await resourceProm;
```

Comment:
Why should we wait for resources to be loaded? It's fine to load those in later too? We can already start showing some UI.